app/models/customer.py

```diff
@@ -0,0 +1,25 @@
+from flask import current_app
+from app import db
+from sqlalchemy.orm import relationship
+
+
+class Customer(db.Model):
+    __tablename__ = "customer"
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.Integer)
+    phone_number = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
```

Comment:
Could you pull videos_checked_out_count from the rental table rather than storing it directly in the customer table?